examples/step-57/doc/intro.dox

```diff
@@ -0,0 +1,324 @@
+\nu_{i+1}|$ is small.  Then we use the solution to the NSE with viscosity
+$\nu_{i}$ as the initial guess of the NSE with $\nu_{i+1}$. It is like a
+staircase from Stokes equations to the NSE we want to solve.
+
+That is we first solve a Stokes problem
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0
+@f}
+
+to get the initial guess for
+
+@f{eqnarray*}
+- \nu_{1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f},\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+Here $\nu_{1}$ is relatively large so that the solution to Stokes problem with viscosity $\nu_{1}$
+can be used as an initial guess for NSE in Newton's iteraton. 
+
+And then the solution to
+
+@f{eqnarray*}
+- \nu_{i} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0.
+@f}
+
+acts as the initial guess for
+
+@f{eqnarray*}
+- \nu_{i+1} \Delta\textbf{u} + (\textbf{u} \cdot \nabla)\textbf{u} + \nabla p &=& \textbf{f}\\
+- \nabla \cdot \textbf{u} &=& 0
+@f}
+with $|\nu_{i+1} - \nu_{i}|$ small enough. 
+
+We repeat this process till getting the solution to the NSE with viscosity
+$\nu_{n-1}$, then the initial guess for the target NSE is obtained.
+
+<h3>The Solver and Preconditioner </h3>
+
+At each step of Newton's iteration, the problem results in solving a
+saddle point systems of the form
+@f{eqnarray*}
+    \left(
+      \begin{array}{cc}
+        A & B^{T} \\
+        B & 0 \\
+      \end{array}
+    \right)
+    \left(
+      \begin{array}{c}
+        U \\
+        P \\
+      \end{array}
+    \right)
+    =
+    \left(
+      \begin{array}{c}
+        F \\
+        0 \\
+      \end{array}
+    \right).
+@f}
+
+Instead of solving the above system, we can solve the equivalent system
+
+@f{eqnarray*}
+    \left(
+      \begin{array}{cc}
+        A + \gamma B^TW^{-1}B & B^{T} \\
+        B & 0 \\
+      \end{array}
+    \right)
+    \left(
+      \begin{array}{c}
+        U \\
+        P \\
+      \end{array}
+    \right)
+    =
+    \left(
+      \begin{array}{c}
+        F \\
+        0 \\
+      \end{array}
+    \right)
+@f}
+with a parameter $\gamma$ and an invertible matrix W.
+
+Denoting the system matrix of the new system by $G$ and right hand
+side by $b$, we solve it iteratively with right preconditioning
+$P^{-1}$ as $GP^{-1}y = b$, where
+
+@f{eqnarray*}
+P^{-1} = \left(\begin{array}{cc} \tilde{A} & B^T \\ 
+                                 0 & \tilde{S} \end{array}\right)^{-1},
+@f}
+
+with $\tilde{A} = A + \gamma B^TW^{-1}B$ and $\tilde{S}$ is the
+corresponding Schur complement $\tilde{S} = B^T \tilde{A}^{-1} B$. we
+let $W = M_p$ where $M_p$ is the pressure mass matrix, then
+$\tilde{S}^{-1}$ can be approximated by
+
+@f{eqnarray*}
+\tilde{S}^{-1} \approx -(\nu+\gamma)M_p^{-1}.
+@f}
+See "An Augmented Lagrangian-Based Approach to the Ossen Problem,
+M.Benzi and M.Olshanskii" for details.
```

Comment:
You switch between using [1] and giving the full reference. Please use one variant consistently.